api/src/test/java/org/pmiops/workbench/elasticsearch/ElasticFiltersTest.java

```diff
@@ -925,18 +927,50 @@ public void testAgeQuery() {
             new SearchRequest()
                 .addIncludesItem(
                     new SearchGroup()
-                        .addItemsItem(new SearchGroupItem().addSearchParametersItem(ethParam))));
+                        .addItemsItem(new SearchGroupItem().addSearchParametersItem(ageParam))));
     BoolQueryBuilder ageBuilder =
         QueryBuilders.boolQuery()
             .filter(QueryBuilders.termQuery("is_deceased", false))
             .filter(
                 QueryBuilders.rangeQuery("birth_datetime")
-                    .gt(left)
+                    .gte(left)
```

Comment:
Hm - did this change? Or was this a bug?

Reply:
A bug. So participants that have birthdays today wouldn't be included in the left most range.